scalatest/src/main/scala/org/scalatest/run.scala

```diff
@@ -130,7 +130,7 @@ object run {
    * <code>fullStacksPassed</code>, and <code>statsPassed</code>).
    * </p>
    */
-  def apply(suite: Suite, testName: String = null, configMap: ConfigMap = ConfigMap.empty): Unit = {
+  def apply(suite: Suite, testName: String | Null = null, configMap: ConfigMap = ConfigMap.empty): Unit = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.